src/routes/_components/Nav.html

```diff
@@ -48,6 +48,12 @@
     }
   }
 
+  @media (min-width: 992px) {
```

Comment:
After playing with this locally, I would like to tweak this a bit:

```suggestion
  @media (min-width: 1400px) {
```

(I'm following [Bootstrap 5's breakpoints](https://getbootstrap.com/docs/5.0/layout/breakpoints/) BTW.)

The reason is that at 992, it is always left-justified once it gets large enough to show the text rather than the icons. I feel that the center justification is only really necessary on very large screens where it starts to look odd to have so much empty space on the right side of the nav bar.